cli/internal/cloudcmd/tfupgrade.go

```diff
@@ -27,13 +27,13 @@ func planUpgrade(
 		return false, fmt.Errorf("backup directory %s already exists: %w", backupDir, err)
 	}
 
-	// Backup the old Terraform workspace and move the embedded Terraform files into the workspace.
-	err := tfClient.PrepareUpgradeWorkspace(
-		templateDir,
-		backupDir,
-		vars,
-	)
-	if err != nil {
+	// Backup old workspace
+	if err := fileHandler.CopyDir(templateDir, backupDir); err != nil {
```

Comment:
we need to copy the workspace dir (template dir are the newly embedded files) to the backup dir.

Reply:
please add a test case that checks that the files from the working dir are copied to the backup and that new embedded files are copied to the working dir